### PR TITLE
Use latest Quix Streams v2 features

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Update to use latest Quix Streams v2 features.

## Review

* [Quickstart](https://quixdocsdev.blob.core.windows.net/pr286/get-started/quickstart.html)
* [Quix Tour](https://quixdocsdev.blob.core.windows.net/pr286/get-started/quixtour/overview.html)
* [Read CSV](https://quixdocsdev.blob.core.windows.net/pr286/develop/integrate-data/read-csv.html)
* [alerting threshold](https://quixdocsdev.blob.core.windows.net/pr286/tutorials/influxdb-alerting/threshold-detection.html)
* [pager duty alerting](https://quixdocsdev.blob.core.windows.net/pr286/tutorials/influxdb-alerting/add-alerting.html)

### CODE REVIEW

File: docs/get-started/quixtour/process-threshold.md [hunk 1]
```diff
@@ -13,35 +13,39 @@ To create the threshold detection transform:
...
     # Create a StreamingDataFrame to process data
     sdf = app.dataframe(input_topic)
 
-    # Filter messages with "cpu_load" > 20
-    sdf = sdf[sdf["cpu_load"] > 20]
+    # Filter in all rows where CPU load is over 20.
+    sdf = sdf.filter(lambda row: row["cpu_load"] > 20)
 
-    # Print filtered messages to the console
-    sdf = sdf.update(lambda row: print(row))
```
Comment (tomas-quix) [inline docs/get-started/quixtour/process-threshold.md:39]: I would keep update, maybe after the last apply so they see it is working.

File: docs/get-started/quixtour/serve-sms.md [hunk 2]
```diff
@@ -55,21 +53,19 @@ To create the SMS alert destination:
...
     def send_alert(row):
         """
         Trigger a CPU spike alert and send an SMS notification
         """
-        cpu_load = row['cpu_load']
-        print("Warning! CPU spike of {} detected.".format(cpu_load))
-        msg = f"Warning! CPU spike of {cpu_load} detected."
         if SEND_SMS:
-            send_sms(msg)    
+            send_sms(row["alert"]["message"])
```
Comment (tomas-quix) [inline docs/get-started/quixtour/serve-sms.md]: Don't even do this, just send the row. If we don't like JSON, just send a string in the alerting message. Sinks should have zero business logic.

File: docs/get-started/quixtour/serve-sms.md [hunk 3]
```diff
@@ -84,10 +80,6 @@ To create the SMS alert destination:
     # Trigger the "send_alert" function for each incoming message
     sdf = sdf.update(send_alert)
 
-    # Print incoming messages to the console
-    sdf = sdf.update(lambda row: print(row))
```
Comment (tomas-quix) [inline docs/get-started/quixtour/serve-sms.md:88]: I would maybe keep it in some capacity?

File: docs/tutorials/influxdb-alerting/add-alerting.md [hunk 4]
```diff
@@ -33,7 +35,7 @@ To add a PagerDuty alerting destination to your pipeline:
             "payload": {
```
Comment (tomas-quix) [inline docs/tutorials/influxdb-alerting/add-alerting.md]: Don't be afraid to just paste the payload directly from the row. Do a validation if you need to and raise an exception if message is not what is expected.

File: docs/get-started/quixtour/process-threshold.md (hunk 1 above)
Comment (tbedford) [reply docs/get-started/quixtour/process-threshold.md:39]: update added after apply

File: docs/get-started/quixtour/serve-sms.md (hunk 2 above)
Comment (tbedford) [reply docs/get-started/quixtour/serve-sms.md]: Changed to just send row.

File: docs/get-started/quixtour/serve-sms.md (hunk 3 above)
Comment (tbedford) [reply docs/get-started/quixtour/serve-sms.md:88]: Reinstated

File: docs/tutorials/influxdb-alerting/add-alerting.md (hunk 4 above)
Comment (tbedford) [reply docs/tutorials/influxdb-alerting/add-alerting.md]: Changed so that sink accepts a row, which is used as the payload of the PagerDuty alert.

**Pull request merged**